src/typography/print/index.md

```diff
@@ -8,5 +8,5 @@ title: Print
 The recommended sizes for common document formats are illustrated here.
 {% sectionHighlight { classes: "dark-magenta" } %}
 
-![TODO](./print-guides.svg)
+![Diagram of A4-sized letterhead and letter. Labels show minimum sizes for the logo, header and body text.](./print-guides.svg)
```

Comment:
The most important information is missing here. And this is one of those things that needs to be in the body copy.
That the minimum height for the logo should be 10mm, the minimum size for the header text should be 16pt and for the body text 12pt.